src/ui/components/results/verticalresultscomponent.js

```diff
@@ -254,9 +255,17 @@ export default class VerticalResultsComponent extends Component {
     };
   }
 
-  mount () {
+  mount (container) {
     if (Object.keys(this.getState()).length > 0) {
-      super.mount();
+      const searchState = this.getState('searchState');
+      const camelCase = (str) => {
+        return str
+          .split('-')
+          .reduce((a, b) => a + b.charAt(0).toUpperCase() + b.slice(1));
+      };
+      Object.entries(SearchStates).forEach(([k, state]) => super.removeContainerClass(`yxt-Results--${camelCase(state)}`));
```

Comment:
is there a reason we use a different for loop b/t here and universalresults?

Reply:
Oops, universal results is wrong. I'll update